src/components/input/link.module.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { PropsWithChildren } from "react";
+import styles from "./button.module.scss";
+import NextLink from "next/link";
+
+
+const Link: React.FC<PropsWithChildren<Props>> = ({ href,
+  children,
+  size = "normal",
+  variant = "primary",
+  onClick = () => undefined,
```

Comment:
What's the use case behind a required `href` and an optional click handler? What would the handler do for example